examples/with-mdx/package.json

```diff
@@ -8,6 +8,7 @@
   },
   "dependencies": {
     "@mdx-js/loader": "^1.5.1",
+    "@mdx-js/react": "^1.6.18",
```

Comment:
Required until https://github.com/mdx-js/mdx/pull/1300 is merged

Reply:
Seems they said they wontfix.

Reply:
Yeah, it's a shame they decided to go that route. Added it as a peer dependency to `@next/mdx` as that is essentially what it is.